app/frontend/actions.ts

```diff
@@ -34,10 +34,24 @@ import {State, GetStateFn, SetStateFn, getSourceAccountInfo} from './state'
 import ShelleyCryptoProviderFactory from './wallet/shelley/shelley-crypto-provider-factory'
 import {ShelleyWallet} from './wallet/shelley-wallet'
 import {parseUnsignedTx} from './helpers/cliParser/parser'
-import {TxPlan, unsignedPoolTxToTxPlan} from './wallet/shelley/shelley-transaction-planner'
+import {
+  TxPlan,
+  TxPlanResult,
+  TxPlanResultType,
+  unsignedPoolTxToTxPlan,
+} from './wallet/shelley/shelley-transaction-planner'
 import getDonationAddress from './helpers/getDonationAddress'
 import {localStorageVars} from './localStorage'
-import {AccountInfo, Ada, Lovelace, CryptoProviderFeature, _Address, TxType, AuthMethodType} from './types'
+import {
```

Comment:
Tooling: There is an eslint plugin that converts type imports to `import type {TypeA, ...}` format which is quite preferred. We should set it up...